ironfish-cli/src/commands/wallet/transactions/delete.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { NodeUtils, TransactionStatus } from '@ironfish/sdk'
+import { Args, ux } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+
+export default class TransactionsDelete extends IronfishCommand {
+  static description = 'delete an expired or pending transaction from the wallet'
+
+  static args = {
+    transaction: Args.string({
+      required: true,
+      description: 'Hash of the transaction to delete from the wallet',
+    }),
+  }
+
+  static flags = {
+    ...RemoteFlags,
```

Comment:
nit: we should not use RemoteFlags for this command since it's local only at the moment. We should convert it to use an RPC later, though.